src/status_im/ui/screens/views.cljs

```diff
@@ -73,7 +72,7 @@
     (when view-id
       (let [current-view (validate-current-view view-id signed-up?)]
         (let [component (case current-view
-                          (:wallet :chat-list :discover :contact-list) main-tabs
+                          (:wallet :chat-list :my-profile) main-tabs
```

Comment:
Shall we rename `:chat-list` to `:home` for clarity?

Reply:
hm, good question, decided to don't do this in this PR, but maybe you're right better to rename for clarity